addons/stock/views/stock_quant_views.xml

```diff
@@ -180,12 +180,6 @@
         <field name="context">{'search_default_internal_loc': 1, 'search_default_productgroup':1, 'search_default_locationgroup':1}</field>
         <field name="res_model">stock.quant</field>
     </record>
-    <record model="ir.actions.act_window" id="location_open_quants"> <!-- Used in location -->
-        <field name="context">{'search_default_productgroup': 1}</field>
-        <field name="domain">[('location_id', 'child_of', active_ids)]</field>
-        <field name="name">Current Stock</field>
-        <field name="res_model">stock.quant</field>
-    </record>
```

Comment:
I think maybe you can do a simple one line solution since in v14 this list view is **not** editable :thinking:. I think since the refactor in v15 added the new view `stock.quant.inventory.tree.editable`, it is being called by this action. You should be able to set its `view_id`  to `stock.quant.tree`, which is the view it was previously using. If the customer really wants to be able to edit products qtys by location, they can still do so using the Inventory Adjustments grouped/filtered by Location or with Reporting > Inventory Report.

TLDR: we can probably minimize the stable changes since the user can still do the same word in other views.